modules/swagger-models/src/main/java/io/swagger/models/refs/GenericRef.java

```diff
@@ -1,5 +1,7 @@
 package io.swagger.models.refs;
 
+import org.apache.commons.lang3.StringUtils;
+
```

Comment:
We try not to add deps if not strictly necessary, use instead indexOf / charAt directly in code